frontend/docs/admin-guide/install.md

```diff
@@ -18,82 +18,82 @@ In order to build and deploy {{ $params.APPLICATION_NAME }} to your AWS account
+5. The role has a default name that you cannot change. You can update the description if you want and then click "Create role"
 
 ### Application Roles
 
 We generally expect customers to use their own roles for the {{ $params.APPLICATION_NAME }} APIGW and lambda execution role as well as for the default notebook role.
 While customers may scope these roles down based on the guidelines of their own organization, the following can be used to quickly stand up
-an instance of {{ $params.APPLICATION_NAME }} for demo purposes only. As written these roles and policies should not be used for production usecases.
+an instance of {{ $params.APPLICATION_NAME }} for demo purposes only. As written, these roles and policies should not be used for production use cases.
 
 The policies below include a number of placeholder variables that you'll need to replace. These policies are meant to serve as a starting point and are tightly scoped
-to the resources {{ $params.APPLICATION_NAME }} expects to use, you can relax these restrictions as necessary
+to the resources {{ $params.APPLICATION_NAME }} expects to use. You can relax these restrictions as necessary
 or make any additional changes required for your environment.
 
 | Variable | Expected Value | Example |
-|--|--|--|
-|`{AWS_PARTITION}`| The partition into which {{ $params.APPLICATION_NAME }} is being deployed. | `aws` |
-|`{AWS_REGION}`| The region into which {{ $params.APPLICATION_NAME }} is being deployed. | `us-east-1` |
-|`{AWS_ACCOUNT}`| The account number of the account into which {{ $params.APPLICATION_NAME }} is being deployed. | `123456789012` |
-|`{MLSPACE_KMS_KEY_ID}`| The id of the KMS Key which is being used to encrypt data in {{ $params.APPLICATION_NAME }}. | `eeecf0d8-44f3-4b29-8b78-55de1b5dc153` |
-|`{MLSPACE_PRIVATE_SUBNET_1}`| The subnet id of one of the {{ $params.APPLICATION_NAME }} VPC private subnets. | `subnet-0a11b2c3333dd44e5` |
-|`{MLSPACE_PRIVATE_SUBNET_2}`| The subnet id of one of the {{ $params.APPLICATION_NAME }} VPC private subnets. | `subnet-0a11b2c3333dd44e5` |
-|`{MLSPACE_PRIVATE_SUBNET_3}`| The subnet id of one of the {{ $params.APPLICATION_NAME }} VPC private subnets. | `subnet-0a11b2c3333dd44e5` |
-|`{MLSPACE_VPC_SECURITY_GROUP}`| The id of the default security group for the {{ $params.APPLICATION_NAME }} VPC. | `sg-903004f8` |
-|`{EMR_DEFAULT_ROLE_ARN}` | The ARN of the role that will be used as the "ServiceRole" for all EMR Clusters created via {{ $params.APPLICATION_NAME }} | `arn:aws:iam::123456789012:role/EMR_DefaultRole`|
-|`{EMR_EC2_INSTANCE_ROLE_ARN}` | The ARN of the role that will be used as the "JobFlowRole" and "AutoScalingRole" for all EMR Clusters created via {{ $params.APPLICATION_NAME }} | `arn:aws:iam::123456789012:role/EMR_EC2_DefaultRole`|
-| `{MLSPACE_APP_ROLE_NAME}` | The name of the {{ $params.APPLICATION_NAME }} application role | `mlspace-app-role` |
+|----------|----------------|---------|
```

Comment:
does the number of `-` matter here? as long as you confirmed the table still looks good, im ok with it.

Reply:
It has no effect. Just the LLM being pedantic.